CONTRIBUTING.md

```diff
@@ -87,7 +87,6 @@ This Wiki is intended to be for _anyone_ -- regardless of expertise -- who is in
 
 - Fork this repository in Github's web interface by pressing the fork icon.
 - Clone your fork and `cd` into the new directory: `git clone https://github.com/username/wiki && cd wiki`
-- Check out the "dev" branch: `git checkout -b dev`
```

Comment:
- checkout a new branch with a descriptive name for your work. `git checkout -b ha-presence-detectiion-with-monitor` This is informative as it tells reviewers a little about what the branch will be committing.

Reply:
i wrote this some time ago now but if i remember this step is just checking out the branch to your local PC i did not think you could add descriptive names till you check it back in and add a comment? again i could be missing something

Reply:
After you clone the repo to your local machine. You would normally create a working branch. This is where you do your edits. It protects the dev branch. And allows you to pull in any upstream updates. This way you can hopefully avoid and fix merge conflicts. 

At least that is my usually work practice.

Reply:
For this particular repo. 'Dev' is the default branch. So there is no real need to do a checkout.

Reply:
> 
> 
> After you clone the repo to your local machine. You would normally create a working branch. This is where you do your edits. It protects the dev branch. And allows you to pull in any upstream updates. This way you can hopefully avoid and fix merge conflicts.
> 
> At least that is my usually work practice.

understand but in this case are they not already working on there own repo?

Reply:
> 
> 
> For this particular repo. 'Dev' is the default branch. So there is no real need to do a checkout.

ya i agree but thats how this repo is setup others may not be i was trying to make the steps more generic but of now its fine i can add back in if we get any more feed back on it for now im just going to approval all the changes. thanks!!

Reply:
They are working on their own repo. That is true.

But let's say you wanted to work on creating two seperate articles at the same time. I don think it would be possible to keep them as separate pull requests easily. You would need to finish both before they could be merged.
Having something like:
dev->new-branch-a
dev->new-branch-b
 
This would allow you to work on two articles at the same time. Without them effecting each other too much. The branch name also keep it clear what you are working on.

I think it makes it easier to keep you local 'dev' and GitHub 'dev' in sync with the upstream 'dev' located under self-hosted-show/wki

pull selfhosted/wiki/ -> WhiteNight121/wiki
pull origin(WhiteNIght121) -> local machine

If that makes sense.

Reply:
Keep in mind this just my subjective opinion. :)

Reply:
sort of but also i dont think im at that level with git yet... but good to think about...